pathfinder/paizo/player_companion/blood_of_the_elements/bote_abilities.lst

```diff
@@ -0,0 +1,84 @@
+
+
+# ==============================
+# New Regional Affinity
+# ==============================
+
+City of Brass				NAMEISPI:YES			CATEGORY:Background		TYPE:Region		VISIBLE:DISPLAY		DESC:Surrounded by seas of flame and magma, the City of Brass is an oasis of civilization within an otherwise brutal realm.	TEMPLATE:Regional Affinity (City of Brass)				SOURCEPAGE:p.27
+Eye of Abendego				NAMEISPI:YES			CATEGORY:Background		TYPE:Region		VISIBLE:DISPLAY		DESC:A permanent hurricane off the coast of the Sodden Lands.																TEMPLATE:Regional Affinity (Eye of Abendego)			SOURCEPAGE:p.13
+Plane of Air				NAMEISPI:YES			CATEGORY:Background		TYPE:Region		VISIBLE:DISPLAY		DESC:A seemingly endless expanse of sky.																					TEMPLATE:Regional Affinity (Plane of Air)				SOURCEPAGE:p.19
+Plane of Earth				NAMEISPI:YES			CATEGORY:Background		TYPE:Region		VISIBLE:DISPLAY		DESC:There is no sky, only earth above and earth below.																		TEMPLATE:Regional Affinity (Plane of Earth)				SOURCEPAGE:p.21
+Plane of Fire				NAMEISPI:YES			CATEGORY:Background		TYPE:Region		VISIBLE:DISPLAY		DESC:A realm with oceans of flame, rivers of molten metal, and mountain ranges composed of the ash.							TEMPLATE:Regional Affinity (Plane of Fire)				SOURCEPAGE:p.23
+Plane of Water				NAMEISPI:YES			CATEGORY:Background		TYPE:Region		VISIBLE:DISPLAY		DESC:A nigh-infinite extraplanar ocean of primordial water.																	TEMPLATE:Regional Affinity (Plane of Water)				SOURCEPAGE:p.25
+
+
+
+# ==============================
+# Cavalier Order
+# ==============================
+
+# Order of the Flame
+Order of the Flame						KEY:Order of the Flame ~ Cavalier Order				CATEGORY:Special Ability	TYPE:CavalierOrder.SpecialQuality								DESC:A cavalier of the order of the flame devotes his life to the pursuit of personal glory. Even if such pursuit puts his own life on the line, the cavalier pursues legendary status and the fame and glory associated with such renown.						ABILITY:Special Ability|AUTOMATIC|Order of the Flame ~ Edicts|PREVARGTEQ:OrderAbilityLvl,1		ABILITY:Special Ability|AUTOMATIC|Order of the Flame ~ Challenge|PREVARGTEQ:OrderAbilityLvl,1			ABILITY:Special Ability|AUTOMATIC|Order of the Flame ~ Skills|PREVARGTEQ:OrderAbilityLvl,1			ABILITY:Special Ability|AUTOMATIC|Order of the Flame ~ Order Abilities|PREVARGTEQ:OrderAbilityLvl,1				ABILITY:Special Ability|AUTOMATIC|Order of the Flame ~ Foolhardy Rush|PREVARGTEQ:OrderAbilityLvl,2				ABILITY:Special Ability|AUTOMATIC|Order of the Flame ~ Daunting Success|PREVARGTEQ:OrderAbilityLvl,8	ABILITY:Special Ability|AUTOMATIC|Order of the Flame ~ Blaze of Glory|PREVARGTEQ:OrderAbilityLvl,15			SOURCEPAGE:p.7	
```

Comment:
Naming convention - Should be "Cavalier Order ~ Order of the Flame".
I corrected a few of these earlier today in Ultimate Combat